.cirrus.yml

```diff
@@ -40,7 +40,7 @@ meta_task:
     skip: $CIRRUS_BRANCH =~ ".*\.tmp"
 
     container:
-        image: "quay.io/libpod/imgts:${IMAGE_SUFFIX}"
+        image: "quay.io/libpod/imgts:latest"
```

Comment:
Can we pin it to some non-latest tag?  I have strong feelings on using latest.  If it bubbles into a stable branch, updating the image has some potential of breaking these stable branches.

Reply:
Typically I agree, and there's certainly a danger of a new broken `imgts` builds breaking everything.  However, our own CI is the only consumer of this container, and new builds are tested before being pushed to latest.  We also have fairly robust monitoring in place to notice if `imgts` fails on a branch we care about.  So I decided that the cost of additional build-automation complexity is probably not worth it.  Should hindsight proves me wrong, I'll just have to eat my hat and open up 13-trillion PRs again to move to tagged versions.